ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/zscaler_endpoints.schema.yml

```diff
@@ -0,0 +1,68 @@
+# Copyright (c) 2023-2024 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  zscaler_endpoints:
+    type: dict
+    documentation_options:
+      table: wan-internet-exit-settings
```

Comment:
remove this - make a this in its own table for Zscaler with a info about the loolup plugin